Core/clim-basic/decls.lisp

```diff
@@ -988,6 +992,8 @@ panes."))
     (frame input-context stream x y &key event))
 (defgeneric frame-input-context-button-press-handler
     (frame stream button-press-event))
+(defgeneric frame-input-context-track-pointer
+    (frame input-context stream event))
```

Comment:
Maybe add a comment to the effect that this is a non-standard extension.

Reply:
I think that this is implicit from the clim package, from the clim-internals perspective it doesn't matter